pilot/pkg/networking/core/v1alpha3/httproute_test.go

```diff
@@ -731,63 +795,25 @@ func TestSidecarOutboundHTTPRouteConfig(t *testing.T) {
 				"test-svc.testns.svc.cluster.local:80": {
 					"test-svc.testns.svc.cluster.local": true, "test-svc.testns.svc.cluster.local:80": true,
 				},
+				"block_all": {
+					"*": true,
+				},
 			},
+			registryOnly: true,
 		},
 	}
 
 	for _, c := range cases {
 		t.Run(c.name, func(t *testing.T) {
 			testSidecarRDSVHosts(t, services, c.sidecarConfig, c.virtualServiceConfigs,
-				c.routeName, c.expectedHosts, c.fallthroughRoute, c.registryOnly)
+				c.routeName, c.expectedHosts, c.registryOnly)
 		})
 	}
 }
 
-func TestSidecarOutboundHTTPRouteConfigWithFallthroughRouteEnabled(t *testing.T) {
```

Comment:
This seems like you are just removing a test without replacing it? am I missing something?

Reply:
Now all the tests are with FallthroughRouteEnabled, so in a way we are testing this in all the tests.